src/main/scala/vectorpipe/osm/internal/PlanetHistory.scala

```diff
@@ -111,14 +114,19 @@ private[vectorpipe] object PlanetHistory {
 
               /* The user who changed the Nodes in this timeslice is credited
                * with the creation of the `Line`, even if they weren't the one
-               * who created the Way to begin with.
+               * who created the Way to begin with (unless the Node previously
+               * existed, in which case the Way's author should be credited).
```

Comment:
Ok, point taken.

Reply:
Without this, I was seeing geometries being incorrectly credited.